pages/[sectionSlug]/index.vue

```diff
@@ -35,7 +35,7 @@ onMounted(() => {
         :tags="[
           {
             name: article.section.name,
-            slug: article.section.slug,
+            slug: `/tags/${article.section.slug}`,
```

Comment:
making sure it is pointing to the root/tags/....

I did this in a new places that needed it.